src/Bundle/Builder/Filter/Filter.php

```diff
@@ -29,7 +29,7 @@ final class Filter implements FilterInterface
 
     private array $formOptions = [];
 
-    private array $criteria = [];
+    private mixed $criteria = [];
```

Comment:
oh... you're right, criteria in Definition Filter is mixed.
Normally this is not a bc change, I've to check

Reply:
maybe null instead of []...

Reply:
that does not work...
https://3v4l.org/ijVvU

We have to define a new setDefaultValue and getDefaultValue instead I think.
And maybe that's better.